extensions/positron-python/python_files/positron/positron_ipykernel/tests/test_inspectors.py

```diff
@@ -42,11 +43,6 @@
 )
 from .utils import get_type_as_str
 
-try:
```

Comment:
torch is well supported for 3.12 now, removing old code 😬

Reply:
What do you think is a good strategy to avoid these kinds of skips being kept around too long moving forward? Open an issue and putting it in a milestone a bit in the future?

Reply:
The good thing about these is that they are pretty benign; that is, once we're at a state where `x_package` is installed, the test will automatically start running.

I think an issue in a milestone would be helpful! I bet some sort of standard overlay/comment to help us search for what needs to be updated would be helpful too. Even if it's something as simple as `# TODO: 3.13 maint` or similar.